### PR TITLE
QA-235: pipeline: Switch to static checks by golangci-lint

### PR DESCRIPTION
**Pull request by lluiscampos**
Modify the existing .golangci.yml to follow standard and fix one linting
issue.

### CODE REVIEW

**Comment by lluiscampos** [comment]
@mender-test-bot start pipeline :one::two:

**Review by tranchitella — CHANGES_REQUESTED** [review]
We cannot remove migration 1.0.0, if we are not using it, it's a bug and we have to fix it.

**Comment by lluiscampos** [comment]
> We cannot remove migration 1.0.0, if we are not using it, it's a bug and we have to fix it.

I suspected that.

It has been "fixed" in #4, so I'll rebase and rework :smiley:

**Comment by lluiscampos** [comment]
@tranchitella Fixed now. All linting issues were fixed by @alfrunes in #4, so this PR now is basically replacing the template :+1:

File: .golangci.yml [hunk 1]
```diff
@@ -43,4 +43,4 @@ linters-settings:
     # max line length, lines longer will be reported. Default is 120.
     line-length: 100
     # tab width in spaces. Default to 1.
-    tab-width: 8
+    tab-width: 4
```
Comment (alfrunes) [inline .golangci.yml:46]: :see_no_evil:

**Review by alfrunes — APPROVED** [review]

**Review by tranchitella — APPROVED** [review]

**Pull request merged**